selenium-connect.gemspec

```diff
@@ -14,10 +14,10 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>=1.9'
 
-  s.add_dependency 'selenium-webdriver', '2.53.1'
+  s.add_dependency 'selenium-webdriver'
   s.add_dependency 'rake'
-  s.add_dependency 'sauce'
+  s.add_dependency 'sauce', '~> 3.7'
   s.add_dependency 'sauce-connect'
   s.add_dependency 'sauce_whisk'
-  s.add_dependency 'appium_lib', '~> 6.0.0'
+  s.add_dependency 'appium_lib', '~> 8.0.0'
```

Comment:
That is a good point, they may well not. This was really just some acrobatics to work around appium not being updated to use selenium 3. I just checked and it is now, so I can simplify that. As for the ios tests, I don't want to let them hold us back, so what I'll do is set a permanent tag for v3.7.1 of selenium connect, and not update test selenium ios.